pkg/process/process.go

```diff
@@ -46,6 +46,7 @@ type ProcessInternal struct {
 	// garbage collector metadata
 	color  int
 	refcnt uint32
+	nsPid  uint32
```

Comment:
nit: Can you please move that after `namespaces`? This is more an "additional internal field" rather than "garbage collector metadata".

Reply:
ok